code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -265,17 +278,25 @@
 			reagents.trans_to_obj(P,amount_per_pill)
 			if(src.loaded_pill_bottle)
 				loaded_pill_bottle.insert_into_storage(P)
+		CHEMMASTER_DISPENSE_SOUND
 		return TRUE
 
 	else if(action == "pill_sprite")
 		pillsprite = sanitizeSafe(params["pill_sprite"])
+		CHEMMASTER_CHANGESETTINGS_SOUND
 		return TRUE
 
 	else if(action == "bottle_sprite")
 		bottlesprite = sanitizeSafe(params["bottle_sprite"])
+		CHEMMASTER_CHANGESETTINGS_SOUND
 		return TRUE
 	return TRUE
 
+#undef CHEMMASTER_DISPENSE_SOUND
+#undef CHEMMASTER_CHANGESETTINGS_SOUND
+#undef CHEMMASTER_SWITCH_SOUND
+#undef CHEMMASTER_BOTTLE_SOUND
```

Comment:
The undefs should be placed at the bottom of the file unless there is a good reason not to do so.